interface/new/new_comprehensive.php

```diff
@@ -78,22 +78,14 @@ function getSearchClass($data_type) {
 ?>
 <html>
 <head>
-<?php html_header_show(); ?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.